subprojects/parseq-test-api/src/main/java/com/linkedin/parseq/ParSeqUnitTestHelper.java

```diff
@@ -203,6 +203,18 @@ public ScheduledExecutorService getScheduler() {
     return runAndWaitForPlanToComplete("runAndWaitForPlanToComplete", task, time, timeUnit);
   }
 
+  public <T> T runAndWaitForPlanToComplete(Task<T> task) {
```

Comment:
Please add javadoc for all these pubic methods, also be consistent with any other javadoc format in this file.

Reply:
Added javadoc in a new commit.